README.md

```diff
@@ -111,6 +111,70 @@ sqsConsumer.stop();
 -   Throwing an error (or returning a rejected promise) from the handler function will cause the message to be left on the queue. An SQS redrive policy can be used to move messages that cannot be processed to a dead letter queue.
 -   By default messages are processed by 10 at a time – a new batch won't be received until the previous one is processed. To adjust number of messages that is being processed in parallel, use the `batchSize` option detailed below.
 
+## Usage with Lambda
```

Comment:
I'd consider extracting this part to a different readme.
The readme is big enough as it is.

Reply:
can u give me an example? I'm not sure how to structure it (create a `docs` folder, have a backlink to the main readme?).